services/metadata/backend/data/societesavoie.json

```diff
@@ -0,0 +1,338 @@
+        ],
+        "alternativeNames": [
+            {
+                "fr": "societe-savoie"
+            }
+        ],
+        "contactPoint": "http://ns.dasch.swiss/repository#dsp-0121-person-000",
+        "dataManagementPlan": {
+            "__type": "DataManagementPlan",
+            "available": true
+        },
+        "description": {
+            "fr": "Le comté puis duché de Savoie était au Moyen  ge une principauté d’Empire qui présentait la particularité de se trouver aux confluents de trois réalités politiques et culturelles distinctes : l’espace franco-bourguignon, les États du nord de l’Italie et les villes de la Suisse occidentale. La colonne vertébrale de la cour de Savoie était constituée de l’Hôtel, qui regroupait les officiers et les serviteurs devant veiller, en premier lieu, aux besoins domestiques du prince et de sa famille, qu’il s’agisse de les sustenter, de veiller à leurs déplacements, de les vêtir ou encore d’assurer leur sécurité. Dès le XIIIe siècle, l’Hôtel assuma également un rôle politique qui ne cessa de croître, car c’est en son sein que les princes choisissaient ceux dont ils s’entouraient pour exercer le pouvoir, notamment leurs conseillers, leurs ambassadeurs ou encore leurs chefs militaires. Par ailleurs, l’Hôtel revêtait des fonctions de représentation sociopolitiques et symboliques essentielles. L’étude de l’Hôtel, qui s’érige progressivement en institution gouvernementale, s’avère indispensable pour saisir les mécanismes de la formation de l’État princier savoyard.\n\nLe premier objectif du projet consiste en une étude complète des Hôtels de la cour de Savoie – celui du prince, celui de la princesse et celui de leurs enfants. En effet, dès la fin du XIIIe siècle, la comtesse et les enfants du couple princier disposaient de leur propre Hôtel. \nCette recherche porte sur le temps long, du XIIIe au XVe siècle, période décisive pour la genèse de l’État moderne. Il s’agit ainsi de reconstruire, pour la première fois de manière globale, par le biais de sources comptables, normatives et notariales, l’organisation des Hôtels de Savoie, ainsi que leur lente évolution en une institution réglementée, hiérarchisée et codifiée. Les coûts des Hôtels ainsi que leur impact sur les finances savoyardes seront également définis. Le deuxième objectif vise à réaliser une base de données prosopographique des membres des Hôtels. Ceux-ci n’étaient pas qu’une institution : ils étaient composés d’hommes, de femmes et d’enfants appartenant à une microsociété dont les contours précis restent encore à définir. Cette société de cour est abordée dans son ensemble, afin d’apporter une contribution essentielle à la sociologie des élites et de mettre en évidence le poids des réseaux de solidarité et des alliances au sein de la cour de Savoie, mais également de reconstituer des carrières et de saisir l’ascension des officiers et des serviteurs ne faisant pas partie de la noblesse."
```

Comment:
there are these `\n`s in this string. Do they render correctly in the metadata browser locally? If not, you could try `<br/>` instead, but that would be a bit hacky again.

Reply:
thx for the hint!
both did not work, so I omitted both options